lte/gateway/python/magma/pipelined/set_interface_client.py

```diff
@@ -0,0 +1,44 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import grpc
+import logging
+
+from magma.common.service_registry import ServiceRegistry
+from lte.protos.session_manager_pb2 import (
+                                            UPFSessionConfigState)
+from lte.protos.session_manager_pb2_grpc import SetInterfaceForUserPlaneStub
+
+SESSION_SERVICE_NAME = "sessiond"
+
+def set_interface_periodic_session_update (upf_session_config_state: UPFSessionConfigState):
```

Comment:
maybe send_periodic_session_update

Also we should cache the sessiond rpc channel, maybe just pass it as an arg to this function and only lookup if its not passed

Reply:
Comments addressed.